addon/serializers/children.js

```diff
@@ -0,0 +1,12 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serialize: function(snapshot, options) {
```

Comment:
Yes, but `snapshot.modelName` is `children`.  I don't want to use the modelName, I want to override it.